csrc/backend.cpp

```diff
@@ -119,15 +119,34 @@ bool probe_backend(const std::string &backend)
     }
 }
 
-AsyncIO *create_asyncio(unsigned int n_entries, const std::string &backend)
+std::string get_default_backend() {
+    const char* env = getenv("TENSORNVME_BACKEND");
+    if (env == nullptr) {
+        return std::string("");
+    }
+    return std::string(env);
+}
+
+AsyncIO *create_asyncio(unsigned int n_entries, std::string backend)
 {
     std::unordered_set<std::string> backends = get_backends();
+    std::string default_backend = get_default_backend();
+
     if (backends.empty())
         throw std::runtime_error("No asyncio backend is installed");
-    if (backends.find(backend) == backends.end())
-        throw std::runtime_error("Unsupported backend: " + backend);
+
+    if (default_backend.size() > 0) {  // priority 1: environ is set
+        std::cout << "[backend] backend is overwritten by environ TENSORNVME_BACKEND from " << backend << " to " << default_backend << std::endl;
```

Comment:
can we print it only when `TENSORNVME_DEBUG=1`?

Reply:
Added here [`2136188` (#47)](https://github.com/hpcaitech/TensorNVMe/pull/47/commits/21361884d5d9f0a89485fc80009b17b4936b9d74)